it/v3/endpoints/ListLossClaimsControllerISpec.scala

```diff
@@ -69,59 +69,59 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
                                               |    "claims": [
                                               |        {
                                               |            "businessId": "XAIS12345678910",
-                                              |            "typeOfClaim": "carry-forward",
-                                              |            "typeOfLoss": "self-employment",
-                                              |            "taxYearClaimedFor": "2020-21",
-                                              |            "claimId": "000000000000011",
+                                              |            "typeOfLoss": "uk-property-non-fhl",
+                                              |            "typeOfClaim": "carry-sideways",
+                                              |            "taxYearClaimedFor": "2019-20",
+                                              |            "claimId": "AAZZ1234567890A",
                                               |            "sequence": 1,
-                                              |            "lastModified": "2021-07-13T12:13:48.763Z",
+                                              |            "lastModified": "2020-07-13T12:13:763Z",
                                               |            "links" : [
                                               |             {
-                                              |               "href" : "/individuals/losses$uri/000000000000011",
-                                              |               "method": "GET",
-                                              |               "rel": "self"
+                                              |               "href" : "/individuals/losses$uri/AAZZ1234567890A",
```

Comment:
Not your change I realise, but this `uri` which is set to `s"/$nino/loss-claims"` is confusing here. Can you get rid of `uri` and change this to (and similarly for other places where it appears):
```suggestion
                                              |               "href" : "/individuals/losses/$nino/loss-claims/AAZZ1234567890A",
```